src/runtime_src/doc/toc/test.rst

```diff
@@ -15,7 +15,7 @@ downtime provided you use a few scripts we have created:
 Building XRT
 ~~~~~~~~~~~~
 
-Make sure you build XRT on a Centos7.4+ or Ubuntu16.04 or Ubuntu18.04 host.
+Make sure you build XRT on a Ubuntu 20.04 or Ubuntu 22.04 or CentOs 7.9+ or rhel9 host.
```

Comment:
Why only this?
```suggestion
Building XRT is tested on Ubuntu 20.04/22.04, CentOs 7.9+ or RHEL9 host OS.
```
XRT works quite better than you might suggest. :smile:

Reply:
Thank you for the fix, the change was worded poorly!

Reply:
Another small change..
Building XRT is tested on Ubuntu 20.04/22.04, CentOs 7.9+ **and** RHEL9 host OS.

Reply:
Yes you are right, this is better.